instance/instance.go

```diff
@@ -37,6 +37,11 @@ func (s *Store) GetList(w http.ResponseWriter, r *http.Request) {
 	var stateFilterList []string
 	if stateFilterQuery != "" {
 		stateFilterList = strings.Split(stateFilterQuery, ",")
+		if err := models.ValidateStateFilter(stateFilterList); err != nil {
+			log.Error(err, nil)
+			http.Error(w, err.Error(), http.StatusBadRequest)
```

Comment:
This all looks good just wondering if we need to define the swagger spec what the valid states are?

Reply:
The spec does already describe which states are valid, you can find it in the description of the field state